gpkit/interactive/sensitivity_map.py

```diff
@@ -134,15 +132,15 @@ def constraint_latex_list(self, paintby):
                 else:
                     constrs = [constr]
                 constr_texs = []
-                for constr in constrs:
```

Comment:
errr not the argument of the method but rather the iteration variable of the for loop